modules/op-release-notes-1-9.adoc

```diff
@@ -0,0 +1,248 @@
+
+[id="tekton-hub-new-features-1-9_{context}"]
+=== {tekton-hub}
+
+:FeatureName: {tekton-hub}
+include::snippets/technology-preview.adoc[]
+
+* With this update, as an administrator, you can use an external database, such as Crunchy PostgreSQL with {tekton-hub}, instead of using the default {tekton-hub} database. This update helps you to perform the following actions:
+** Specify the coordinates of an external database to be used with {tekton-hub}
+** Disable the default {tekton-hub} database deployed by the Operator
+
+* This update removes the dependency of `config.yaml` from external Git repositories and moves the complete configuration data into the API ConfigMap. This update helps an administrator to perform the following actions:
+** Add the configuration data, such as categories, catalogs, scopes and defaultScopes in the {tekton-hub} custom resource.
+** Modify {tekton-hub} configuration on the cluster. All modifications are preserved upon Operator upgrades.
+** Update the list of catalogs for {tekton-hub}
+** Change the categories for {tekton-hub}
++ 
+[NOTE]
+====
+If you do not add any configuration data, you can use the default data in the API ConfigMap for {tekton-hub} configurations.
+====
+
+[id="pac-new-features-1-9_{context}"]
+=== {pac}
+
+* This update supports concurrency limit in the repository CRD to define the maximum number of `PipelineRuns` for a repository at a time. This update also enhances the `PipelineRun` concurrency to execute pipelines in alphabetical order instead of random.
+
+* This update adds a new command `tkn pac logs` for showing the logs of the latest pipeline run for a repository.
+
+* Before this update, the {pac} controller required to restart to take configuration changes into account. With this update, the {pac} controller takes changes into account based on the `pipelines-as-code` configmap.
+
+* This update supports advanced event matching on file path for push and pull requests to GitHub and GitLab. For example, you can use the Common Expression Language (CEL) to run a pipeline only if a path has changed and match each markdown file only in the `docs` directory. 
++
+[source,yaml]
+----
+  ...
+  annotations:
+     pipelinesascode.tekton.dev/on-cel-expression: |
+      event == "pull_request" && "docs/*.md".pathChanged()
+----
+
+* This update adds a new driver to support the Gitea software package.
+
+* With this update, you can reference a remote pipeline in the `pipelineRef:` object using annotations.
+
+* With this update, you can auto-configure new GitHub repositories with {pac}, which sets up a namespace and creates a custom resource for your repository.
+
+* This update provides metrics for pipeline runs with provider information in the user namespace.
```

Comment:
```suggestion
* With this update Pipelines as Code emits metrics for PipelineRuns with provider information.
```

Reply:
can we use `generate` or `provide` in place of `emits`, which sounds more like something is discharged (not a suitable term)?